### PR TITLE
Food materials audit

### PR DESCRIPTION
**Pull request by mightyagrippa**
Tweaking some food materials per #6045. All sandwiches now have "wheat", all nachos now have "junk", etc. All of the jsons in the comestibles\ dir look fine and didn't need changes.

### CODE REVIEW

**Comment by pisskop** [comment]
Part of why some of these didnt have all their various giblies listed is due to game code.


For instance, putting veggy on curry makes it inedible to carnivores.
Wheat on sandwhiches assumes that the bread is both a significant part of the sandwich and made of actual wheat instead of the chafe whitebread, which is glorified sugar.

**Comment by mightyagrippa** [comment]
I'd say that bread is a significant part of any sandwich, but looking at the recipes it is possible to make sandwiches from cornbread. Though at first glance a cornbread sandwich sounds like a terrible idea. So very crumbly. 

Vegetables in curry seem fine for carnivores, there's code for partial edibility where "You pick out the edible parts and throw away the rest."

**Comment by Coolthulhu** [comment]
JSON error on load.

**Comment by mightyagrippa** [comment]
Probably where I mistyped veggy as veggie. I'll change and attempt to make

**Comment by mightyagrippa** [comment]
Built fine and made a changed item successfully.

File: data/json/items/comestibles.json [hunk 1]
```diff
@@ -7697,7 +7697,7 @@
         "nutrition" : 25,
         "description" : "Fluffy and delicious pancakes with real maple syrup, with delicious chocolate baked right in.",
         "price" : 700,
-        "material" : ["wheat", "junk"],
```
Comment (Coolthulhu) [inline data/json/items/comestibles.json]: Chocolate is junk

**Pull request merged**